code/game/objects/items/melee/misc.dm

```diff
@@ -366,16 +365,16 @@
 /obj/item/melee/roastingstick/proc/on_transform(obj/item/source, mob/user, active)
 	SIGNAL_HANDLER
 
-	extended = active
 	inhand_icon_state = active ? "nullrod" : null
-	balloon_alert(user, "[active ? "extended" : "collapsed"] [src]")
-	playsound(user ? user : src, 'sound/weapons/batonextend.ogg', 50, TRUE)
+	if(user)
+		balloon_alert(user, "[active ? "extended" : "collapsed"] [src]")
```

Comment:
```suggestion
		balloon_alert(user, active ? "extended" : "collapsed")
```

you got rid of the name in the balloon alert earlier when you were touching stuff up, but not here. do you think it should be kept or nah?

Reply:
iirc Mothblocks once said they thought the balloon alerts were too long / it wasn't necessary to mention the name in the alert. But I don't know if it's really the case anymore. What do *you* think, do you prefer `extended baton` or just `extended`?

Reply:
Actually, I rescind that question. I'll revert it and change it in a later pr if I remember